Mixpanel/Mixpanel.m

```diff
@@ -1667,16 +1667,17 @@ - (void)notificationController:(MPNotificationViewController *)controller wasDis
     };
 
     if (status && controller.notification.callToActionURL) {
-        MixpanelDebug(@"%@ opening URL %@", self, controller.notification.callToActionURL);
-        BOOL success = [[UIApplication sharedApplication] openURL:controller.notification.callToActionURL];
+        [controller hideWithAnimation:NO completion:^{
```

Comment:
I'd like to bring some attention to `NO` here. I don't have a strict reason why to dismiss without animation...

The previous logic was: if `UIApplication#openURL:` returns `YES` - then dismiss without animation.

With this PR we are opening the url after it got dismissed. I assume that in most cases of using this library `openURL:` will return `YES` because people are going to use this notifications to show something in the same app or to open a Safari or make a phone call... I don't use `UIApplication#canOpenURL:` because of the notes in the documentation about iOS9. Assuming this library is used by a lot of developers in lots of different apps I think we shouldn't rely on `canOpenURL:`.

If the url is `http` or `tel`, then it's probably fine to dismiss the notification panel without animation and then the app goes into background. If the url is within the same app, then it might make sense to dismiss with animation. As I said before I don't have a strict opinion on this. But maybe someone else has ;)